zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -21,14 +21,18 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 import kotlinx.serialization.modules.SerializersModule
 
-class KtBridge internal constructor(
+/**
+ * An outbound channel for delivering calls to the other platform, and an inbound channel for
+ * receiving calls from the other platform.
+ */
+class Endpoint internal constructor(
```

Comment:
Endpoint implies 1-way?

Reply:
Connection? Pipeline?

Reply:
It’s one end of a duplex connection. In TCP terms this is a Socket.